programmers/짝수와-홀수.js

```diff
@@ -0,0 +1,7 @@
+function solution(num) {
+    if(num % 2 === 0){
+        return "Even";
+    }else{
+        return "Odd";
+    }
```

Comment:
```suggestion
    if (num % 2 === 0) {
        return "Even";
    } else {
        return "Odd";
    }
```
가독성을 위해서 제가 고쳐드린 코드를 보고 공백을 어디에 두면 코드가 더 가독성이 좋은지 한번 생각해보면 좋을것같아요.

```js
function solution(num) {
    return num % 2 ? 'Odd' : 'Even';
}
```
아까 궁금해 하셨던 물음표사용해서 문제를 푸는 방법인데 '삼항연산자' 라고 구글에 검색해보시고 공부해보시기 바랍니다. 

또한, 자바스크립트에서 ===와 ==의 차이는 ===가 더 정확한 비교가 가능하다입니다.

![image](https://user-images.githubusercontent.com/58061847/144834762-09d34ec4-4828-4e2b-8a37-2dd5c3936a74.png)